src/dmd/dsymbolsem.d

```diff
@@ -4564,15 +4557,6 @@ private extern(C++) final class DsymbolSemanticVisitor : Visitor
     {
         //printf("NewDeclaration::semantic()\n");
 
-        // `@disable new();` should not be deprecated
-        if (!nd.isDisabled())
-        {
-            // @@@DEPRECATED_2.091@@@
-            // Made an error in 2.087.
-            // Should be removed in 2.091
```

Comment:
Shouldn't this be 2.097 as per [DIP1013](https://github.com/dlang/DIPs/blob/d7a913b0fbfef14584b43aff9adc863b71702507/DIPs/accepted/DIP1013.md)?

Reply:
yes and no. The deprecation is there since v2.080, as verified on [run.dlang](https://run.dlang.io/is/lMVRoS)

> Up to      2.079.1: Success and no output
2.080.1 to 2.086.1: Success with output: onlineapp.d(3): Deprecation: class allocators have been deprecated, consider moving the allocation strategy outside of the class
Since      2.087.1: Failure with output: onlineapp.d(3): Error: class allocators are obsolete, consider moving the allocation strategy outside of the class

The comment was probably added to match DIP1013 but the date are wrong.